### ISSUE TITLE
[Question] Benefits of sorting dependencies

### ISSUE THREAD

**Issue by jisungbin** [square/gradle-dependencies-sorter#9]
Hi! I'm enjoying Square's open-source efforts, and thank you so much for making an awesome open-source this time around too. 

I have one question about this project. What are the benefits of sorting Gradle dependencies? I think you have a reason for planning and developing this project.

I tried googling about it, but I couldn't get the results I wanted, probably because I got the search keywords wrong.

In summary, what are the benefits of sorting Gradle dependencies?

Thank you.

**Comment by bartek-wesolowski** [comment]
Here are some that I can think of:
- it's more difficult to add duplicate dependencies
- it's easier to say if a project has a particular dependency, because you don't have to go through the whole (sometimes very long) list of dependencies
- there are less conflicts in PRs, because without strict ordering people usually add new dependencies at the end of the list

**Comment by jisungbin** [comment]
Hi @bartek-wesolowski. Thanks for sharing your ideas!

**Comment by autonomousapps** [comment]
I agree with https://github.com/square/gradle-dependencies-sorter/issues/9#issuecomment-1337295899

**Comment by autonomousapps** [comment]
I agree with https://github.com/square/gradle-dependencies-sorter/issues/9#issuecomment-1337295899

**Comment by autonomousapps** [comment]
I agree with https://github.com/square/gradle-dependencies-sorter/issues/9#issuecomment-1337295899
